<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>651</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>Digital Sound!</string>
   </property>
   <widget class="QPushButton" name="btn_13">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>40</y>
      <width>60</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>라#(U)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_11">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>60</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>파#(T)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_8">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>150</y>
      <width>65</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>도(K)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_12">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>40</y>
      <width>60</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>솔#(Y)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>65</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>레(S)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>65</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>미(D)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_6">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>150</y>
      <width>65</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>라(H)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_7">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>150</y>
      <width>65</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>시(J)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_10">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>60</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>레#(E)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>65</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>파(F)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>65</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>도(A)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_back">
    <property name="geometry">
     <rect>
      <x>532</x>
      <y>270</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_5">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>150</y>
      <width>65</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string>솔(G)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_9">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>60</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>도#(W)</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
